iOS/Components/Browser/Album Browser/AlbumViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ViewController.swift
+//  Persephone-iOS
+//
+//  Created by Daniel Barber on 2020-3-13.
+//  Copyright © 2020 Dan Barber. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+
+class AlbumViewController: UICollectionViewController {
+  let dataSource = AlbumDataSource()
+  
+  let itemsPerRow: CGFloat = 2
+  
+  let sectionInsets = UIEdgeInsets(
+    top: 20.0,
+    left: 20.0,
+    bottom: 30.0,
+    right: 20.0
+  )
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    albumCollectionView.dataSource = dataSource
+
+    App.store.subscribe(self) {
+      $0.select { $0.albumListState }
+    }
+
+    NotificationCenter.default.addObserver(self, selector: #selector(didConnect), name: .didConnect, object: nil)
+
+    title = "Albums"
+    navigationController?.navigationBar.prefersLargeTitles = true
```

Comment:
FWIW if you're using a storyboard, it should be possible to set both of these properties directly in the storyboard file.

Reply:
Took me a while to find!